website/content/api-docs/system/monitor.mdx

```diff
@@ -8,6 +8,9 @@ description: The `/sys/monitor` endpoint is used to receive streaming logs from
 
 The `/sys/monitor` endpoint is used to receive streaming logs from the Vault server.
 
+If Vault is emitting log messages faster than a receiver can process them, the
+log messages will be truncated.
```

Comment:
```suggestion
some log lines will be dropped.
```

Truncation implies (to me at least) that we send partial log lines, which isn't the case.